pkg/vectorize/round/round.go

```diff
@@ -133,10 +134,17 @@ func roundUint64(xs []uint64, rs []uint64, digits int64) []uint64 {
 	case digits >= 0:
 		return xs
 	case digits > -maxUint64digits:
-		scale := scaleTable[-digits]
+		scale := float64(scaleTable[-digits])
 		for i := range xs {
-			value := (xs[i] + scale/2) / scale * scale //todo(broccoli): please find a better way to round away from zero
-			rs[i] = uint64(value)
+			if xs[i]>>52 != 0 { // so this value cannot be safely cast to float64
```

Comment:
Round of uint should not go into float.   Should be all int operation.   What do you exactly want?

round(123, -1) = 120,                 123/10*10 = 120 + (123%10 >=5 ? 10: 0);   Yes?   What was wrong?

Reply:
yes, should be this way

Reply:
modified